frontends/qt/setup.nsi

```diff
@@ -106,6 +107,23 @@ Section -post SEC0001
     WriteRegStr HKCU "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" UninstallString $INSTDIR\uninstall.exe
     WriteRegDWORD HKCU "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" NoModify 1
     WriteRegDWORD HKCU "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" NoRepair 1
+
+    # Links aopp: URI scheme.
+    # It links it silently if no other app is registered, to not overwrite.
+    # If another app is registered, we ask the user for permission.
+    ReadRegStr $0 HKCU "SOFTWARE\Classes\aopp\shell\open\command" ""
+    ${If} $0 != "${AOPP_EXE}"
```

Comment:
One problem with this is during app upgrade. Users can choose the install location, right. If it's different from the previous install, `AOPP_EXE` won't match. But it's an issue bigger than this PR. I guess it's good as is.

Reply:
Hmm good point. In that case the user is asked, which sounds good enough.